addons/mrp/data/mrp_data.xml

```diff
@@ -2,6 +2,22 @@
 <odoo>
     <data noupdate="1">
 
+        <record id="paperformat_euro_landscape" model="report.paperformat">
+            <field name="name">European A4 landscape</field>
+            <field name="default" eval="True" />
+            <field name="format">A4</field>
+            <field name="page_height">0</field>
+            <field name="page_width">0</field>
+            <field name="orientation">Landscape</field>
+            <field name="margin_top">40</field>
+            <field name="margin_bottom">28</field>
+            <field name="margin_left">7</field>
+            <field name="margin_right">7</field>
+            <field name="header_line" eval="False" />
+            <field name="header_spacing">35</field>
+            <field name="dpi">90</field>
+        </record>
```

Comment:
you could remove this part by handling data-report-orientation the same way data-report-dpi is handled